debug/index.html

```diff
@@ -239,6 +239,7 @@ <h1 class="title">Transmux Analyzer</h1>
       codecsArray = codecs.value.split(',');
 
       mediaSource.addEventListener('sourceopen', function () {
+        mediaSource.duration = 0;
```

Comment:
Setting it to zero is not truthful and could mislead downstream clients. This doesn't seem correct to me.